driver-core/src/main/java/com/datastax/driver/core/DataType.java

```diff
@@ -759,4 +772,16 @@ public String toString() {
             return String.format("'%s'", customClassName);
         }
     }
+
+    /**
+     */
+    public static class DurationType extends CustomType {
+
+        private static final DurationType instance = new DurationType();
+
+        private DurationType() {
+            super(Name.CUSTOM, "org.apache.cassandra.db.marshal.DurationType");
+        }
```

Comment:
Nit: we could override `toString()` here, as this produces a slightly better output, for example in `TableMetadata.toString()` (although the full custom type name is valid and entirely compatible).